### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by oyenugae**
:wave:! GitHub Classroom created this pull request as a place for your teacher to leave feedback on your work. It will update automatically. **Don’t close or merge this pull request**, unless you’re instructed to do so by your teacher.

In this pull request, your teacher can leave comments and feedback on your code. Click the **Subscribe** button to be notified if that happens.

Click the **Files changed** or **Commits** tab to see all of the changes pushed to `master` since the assignment started. Your teacher can see this too.

<details>
<summary>
<strong>Notes for teachers</strong>
</summary>

Use this PR to leave feedback. Here are some tips:

- Click the **Files changed** tab to see all of the changes pushed to `master` since the assignment started. To leave comments on specific lines of code, put your cursor over a line of code and click the blue **+** (plus sign). To learn more about comments, read “[Commenting on a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request)”.
- Click the **Commits** tab to see the commits pushed to `master`. Click a commit to see specific changes.
- If you turned on autograding, then click the **Checks** tab to see the results.
- This page is an overview. It shows commits, line comments, and general comments. You can leave a general comment below.

For more information about this pull request, read “[Leaving assignment feedback in GitHub](https://classroom.github.com/help/leaving-feedback-in-github)”.
</details>

Subscribed: @manny-Thrusting

### CODE REVIEW

File: Main.java [hunk 1]
```diff
@@ -1,21 +1,33 @@
...
+   int DaysDebtorDefaulted = 5;
+   int AmountDebtorPayPerDay = 1000;
+   int AmountPaid = 2000;
+   
    
-   //Declare the variable of the total amount the debtor is to pay, the days the debtor paid for, amount the debtor is left to pay, days the debtor has not payed for
+   int TotalAmountToPay;
+   int DaysDebtorPaid;
+   int AmountLeft;
+   int DaysNotPaid;
```
Comment (oyenugae) [inline Main.java:12]: Good job but your amount should have been in float and you should have used the camelCase way of writing variable names

File: Main.java [hunk 2]
```diff
@@ -1,21 +1,33 @@
...
 
-   // calculate and print total amount the debtor is to pay
+   
+   TotalAmountToPay = DaysDebtorDefaulted * AmountDebtorPayPerDay;
+   System.out.println("The total amount the debtor is to pay is "+TotalAmountToPay);
```
Comment (oyenugae) [inline Main.java:16]: Correct

File: Main.java [hunk 3]
```diff
@@ -1,21 +1,33 @@
...
 
-   // calculate and print the days the debtor paid for
   
-  // calculate and print the amount whose day was not captured because the amount was incomplete
-
-   // calculate and print amount the debtor is left to pay
-
-   // calculate and print days the debtor has not payed for
-
```
Comment (oyenugae) [inline Main.java:17]: Why did you leave all this empty

File: Main.java [hunk 4]
```diff
@@ -1,21 +1,33 @@
...
-   //kindly remove the statement below when you are done with the assignment
-    System.out.println("Hello world!");
```
Comment (oyenugae) [inline Main.java:19]: You should have removed this as the instruction said

File: Main.java [hunk 5]
```diff
@@ -1,21 +1,33 @@
...
+   DaysDebtorPaid = AmountPaid / AmountDebtorPayPerDay;
+   System.out.println("No of days debtor paid for: "+DaysDebtorPaid );
```
Comment (oyenugae) [inline Main.java:20]: Correct

File: Main.java (hunk 1 above)
Comment (manny-Thrusting) [reply Main.java:12]: Should I still change it sir?

File: Main.java [hunk 6]
```diff
@@ -1,21 +1,33 @@
...
+  
+  
+  int AmountNotCovered = AmountPaid % AmountDebtorPayPerDay;
+  System.out.println("The Amount Not Captured is "+AmountNotCovered);
```
Comment (oyenugae) [inline Main.java:24]: Correct

File: Main.java (hunk 4 above)
Comment (manny-Thrusting) [reply Main.java:19]: I did remove it sir

File: Main.java [hunk 7]
```diff
@@ -1,21 +1,33 @@
...
+   
+  AmountLeft = TotalAmountToPay - AmountPaid;
+  System.out.println("Amount the debtor is left to pay is "+AmountLeft);
```
Comment (oyenugae) [inline Main.java:27]: Correct

File: Main.java [hunk 8]
```diff
@@ -1,21 +1,33 @@
...
+  
+  DaysNotPaid = DaysDebtorDefaulted - DaysDebtorPaid;
+  System.out.println("The days the debtor has not payed for: "+DaysNotPaid);
```
Comment (oyenugae) [inline Main.java:30]: Correct

File: Main.java (hunk 1 above)
Comment (oyenugae) [reply Main.java:12]: Nope. Time is over. Just giving you a feedback so that you can understand your score when you get it

**Pull request merged**